packages/validation-manager/src/altmempool/AltMempoolConfig.ts

```diff
@@ -25,15 +27,50 @@ export interface BaseAltMempoolRule {
 }
```

Comment:
RuleException: its nice that javascript allows dynamic typing (string/object), but it makes it harder to parse.
better use optional members. e.g: `exceptions: [ { "role":"account" } ]`